examples/svm/plot_svm_regression.py

```diff
@@ -25,7 +25,7 @@
 # Fit regression model
 svr_rbf = SVR(kernel='rbf', C=100, gamma=0.1, epsilon=.1)
 svr_lin = SVR(kernel='linear', C=100)
-svr_poly = SVR(kernel='poly', C=100, degree=3, epsilon=.1, coef0=1)
+svr_poly = SVR(kernel='poly', C=100, degree=3, epsilon=.1, coef0=0)
```

Comment:
why did you change that back?

Reply:
yea that was a typo. I was trying to see what errors it will produce if `coef0=0`.